controllers/mapper.go

```diff
@@ -206,7 +68,6 @@ type Referable interface {
 
 type K8STypeLookup interface {
 	ResourceForReferable(obj Referable) (*schema.GroupVersionResource, error)
-	KindForReferable(obj Referable) (*schema.GroupVersionKind, error)
```

Comment:
if this is not used anywhere, then please remove the implementation method from the same file.